anchor/programs/sponsor-relayer/src/instructions/initialize_relayer.rs

```diff
@@ -0,0 +1,25 @@
+use anchor_lang::prelude::*;
+
+use crate::{Relayer, Sponsorship};
+
+#[derive(Accounts)]
+pub struct InitializeRelayer<'info> {
+    #[account(mut)]
+    pub authority: Signer<'info>,
+    #[account(init, payer = authority, space = 8 + Relayer::INIT_SPACE, seeds = [b"relayer"], bump)]
+    pub relayer: Account<'info, Relayer>,
```

Comment:
May be obvious, but perhaps label this as relayer_pda ?
Just trying to disambiguate from all the possible interpretations of "relayer".